error.vue

```diff
@@ -67,13 +69,19 @@ const backToHomeCta: Link = {
 
 const backToDocsCta: Link = {
   url: "/documentation",
-  label: "Back to Qiskit Documentation",
+  label: "Back to Qiskit documentation",
   segment: {
     cta: "back-to-documentation",
     location: "error-page",
   },
 };
 
+const isDocumentationError = computed(() => {
```

Comment:
This is just a nice to have, but I personally think that explicitly adding the expected type via generics is a good idea when using computed functions. It's easier to read and understand at a glance what is expected and it's also a safety mechanism for refactors.

```suggestion
const isDocumentationError = computed<boolean>(() => {
```

But it's totally optional.

https://vuejs.org/guide/typescript/composition-api.html#typing-computed